### PR TITLE
fix(queue): limit re-deliveries of failed messages

### PR DESCRIPTION
**Pull request by robfletcher**
Once a failed message has been tried 10 times it gets passed to a handler that will terminate the associated pipeline

### CODE REVIEW

**Comment by robfletcher** [comment]
PTAL @spinnaker/reviewers

File: orca-queue-sqs/src/main/kotlin/com/netflix/spinnaker/config/SqsQueueConfiguration.kt [hunk 1]
```diff
@@ -33,14 +35,21 @@ import org.springframework.context.annotation.*
...
 
-  @Bean
-  open fun amazonSqsClient(awsCredentialsProvider: AWSCredentialsProvider, sqsProperties: SqsProperties): AmazonSQS {
-    return AmazonSQSClient.builder()
+  @Bean open fun amazonSqsClient(
+    awsCredentialsProvider: AWSCredentialsProvider,
+    sqsProperties: SqsProperties
+  ) =
+    AmazonSQSClient
+      .builder()
```
Comment (robzienert) [inline orca-queue-sqs/src/main/kotlin/com/netflix/spinnaker/config/SqsQueueConfiguration.kt]: Is this formatting an IntelliJ setting?

**Comment by robzienert** [comment]
👍

File: orca-queue-sqs/src/main/kotlin/com/netflix/spinnaker/config/SqsQueueConfiguration.kt (hunk 1 above)
Comment (robfletcher) [inline orca-queue-sqs/src/main/kotlin/com/netflix/spinnaker/config/SqsQueueConfiguration.kt]: It's me being a prick

**Pull request merged**